cli/tui/scrollback/native_live_area.go

```
@@ -196,60 +199,64 @@ func nativeLiveAreaFramesEqual(left NativeLiveAreaFrame, right NativeLiveAreaFra
-		}
+		out.WriteString(xansi.CursorPosition(1, liveAreaViewportTopRow(renderedLines, terminalHeight)+index))
 		out.WriteString(xansi.EraseEntireLine)
 	}
-	if renderedLines > 1 {
-		out.WriteString(xansi.CursorUp(renderedLines - 1))
-	}
-	out.WriteString("\r")
 	return out.String()
 }
 
 func liveAreaCursorPlacementSequence(cursor NativeLiveAreaCursor, renderedLines int) string {
```

golangci-lint (2.12.2)

[error] func liveAreaCursorPlacementSequence is unused

(unused)